reference/constraints/Collection.rst

```diff
@@ -223,7 +223,7 @@ you can do the following:
                     - Collection:
                         fields:
                             personal_email:
-                                - Required
+                                - Required:
```

Comment:
Is this proposal the right fix?

Shouldn't this:

```yaml
        - Collection:
            fields:
                personal_email:
                    - Required:
                        - NotBlank: ~
                        - Email: ~
```

be like this instead?

```yaml
        - Collection:
            fields:
                personal_email:
                    - Required: ~
                    - NotBlank: ~
                    - Email: ~
```

Reply:
I did not know about "Required" before (always used NotBlank), but I tried
```
- Required:
    - NotBlank: ~
    - Email: ~
```

And it works, and I based my guess also on the annotation code block
```
@Assert\Required({@Assert\NotBlank, @Assert\Email})
````
Which seems to be, for required, a combination of constraints

But according to this:
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Constraints/Required.php

Your guess seems correct too, as it can be nested

Reply:
The nesting is correct here. We do specify the constraints that are required for each collection entry.